wifiphisher/common/recon.py

```diff
@@ -1,116 +1,60 @@
-"""
-This module handles all the operations regarding locating all the
-available access points
-"""
-
-from __future__ import division
-import threading
-import time
-import logging
+"""his module handles all access point the operations."""
```

Comment:
This module does not handle all access point operations. It is not `accesspoint.py`

Reply:
why do you want this to be changed to?

Reply:
Handles all reconnaissance operations.